.github/ISSUE_TEMPLATE/e-question

```diff
@@ -0,0 +1,13 @@
+---
+
```

Comment:
Do we want to encourage devs to ask questions by raising an issue? SO/Slack seem to be the best way to get questions asked in a timely manner

Reply:
We do not. That why the template tells them to look for info elsewhere